tooling/cli/src/init.rs

```diff
@@ -130,29 +130,41 @@ impl Options {
       )
     })?;
 
-    self.before_dev_command = self
-      .before_dev_command
-      .map(|s| Ok(Some(s)))
-      .unwrap_or_else(|| {
-        prompts::input(
-          "What is your frontend dev command?",
-          Some("npm run dev".to_string()),
-          self.ci,
-          true,
-        )
-      })?;
+    let package_manager =
```

Comment:
we can try to infer the package manager by checking if a yarn.lock/package-lock.json/pnpm-lock.yaml exists: https://github.com/tauri-apps/tauri/blob/008b9f7481db3fcfaff5424e4b25a2399959be17/tooling/cli/src/helpers/npm.rs#L36

Reply:
Thanks and I'll refactor the codes.